libs/prisma-models/src/sql_ext/column.rs

```diff
@@ -168,6 +168,7 @@ where
             TypeIdentifier::Xml => TypeFamily::Text,
             TypeIdentifier::DateTime => TypeFamily::DateTime,
             TypeIdentifier::Bytes => TypeFamily::Bytes,
+            TypeIdentifier::Unsupported => panic!("No unsupported field should reach that path"),
```

Comment:
use `unreachable!("...")` for things that are supposed to be unreachable. Same as panic really but has a slightly nicer error.